server/src/internalClusterTest/java/org/opensearch/cluster/coordination/UnsafeBootstrapAndDetachCommandIT.java

```diff
@@ -105,10 +113,32 @@ private void expectThrows(ThrowingRunnable runnable, String message) {
         assertThat(ex.getMessage(), containsString(message));
     }
 
+    private void ensureReadOnlyBlock(boolean expected_block_state, String node) {
+        ClusterState state = internalCluster().client(node).admin().cluster().prepareState().setLocal(true)
+            .execute().actionGet().getState();
+        boolean current_block_state = state.metadata().persistentSettings().
+            getAsBoolean(Metadata.SETTING_READ_ONLY_SETTING.getKey(), false);
+        if(current_block_state != expected_block_state) {
+            fail(String.format("Read only setting for the node %s don't match. Expected %s. Actual %s", node,
+                expected_block_state, current_block_state));
+        }
+    }
+
+    private void removeBlock() {
+        try{
+            if (isInternalCluster()) {
+                Settings settings = Settings.builder().putNull(Metadata.SETTING_READ_ONLY_SETTING.getKey()).build()  ;
+                assertAcked(client().admin().cluster().prepareUpdateSettings().setPersistentSettings(settings).get());
+            }
+        } catch (Exception ex) {
+            logger.error("Exception while updating cluster read only settings ", ex);
+        }
```

Comment:
why do you need catch here? If it throws an exception that should fail the test right?

Reply:
Agree, let me remove this.